src/video/types.rs

```diff
@@ -5,6 +5,11 @@ use super::video_source_redirect::VideoSourceRedirect;
 use paperclip::actix::Apiv2Schema;
 use serde::{Deserialize, Serialize};
 
+pub const KNOWN_RTP_RAW_FORMATS: [&str; 9] = [
+    "RGB", "RGBA", "BGR", "BGRA", "AYUV", "UYVY", "I420", "Y41B", "UYVP",
+];
+pub const DEFAULT_RAW_FORMAT: &str = "I420";
```

Comment:
Did you look at https://docs.rs/gstreamer-video/latest/gstreamer_video/struct.VideoFormatInfo.html#method.name ?

Where the types are defined here:
https://docs.rs/gstreamer-video/latest/gstreamer_video/enum.VideoFormat.html

```
let info = VideoFormatInfo::from_format(crate::VideoFormat::RGB);
dbg!(info)'
```